### PR TITLE
Create central-pipeline.yml and add bors capability

### PR DESCRIPTION
**Pull request by glwagner**
This PR starts transferring CI from tartarus and sverdrup to Caltech's central cluster, and implements `bors` trying and staging functionality for merging PRs. With @jakebolewski .

### CODE REVIEW

**Comment by glwagner** [comment]
bors try

**Comment by glwagner** [comment]
bors try

**Comment by glwagner** [comment]
bors try

**Comment by glwagner** [comment]
bors try

**Comment by glwagner** [comment]
bors try

**Comment by glwagner** [comment]
bors try

**Comment by glwagner** [comment]
bors try

File: .buildkite/central-pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      slurm_ntasks: 1
+      slurm_gres: "gpu:1"
+
+  - wait
+  
+# "slurm_ntasks: 1" is converted to "-ntasks=1"
+  - label: ":flower_playing_cards: unit tests"
+    key: "gpu_unittests"
+    command:
+      - "julia --color=yes --project=test test/runtests.jl"
```
Comment (jakebolewski) [inline .buildkite/central-pipeline.yml:44]: @glwagner you need to remove `--project=test` here, then I think it should be fine.  I'll put in a ticket to get OpenMPI compiled against CUDA 11.2.

**Comment by glwagner** [comment]
This is stale... maybe we will revisit this some time in the future.

**Pull request closed without merging**